.cirrus.yml

```diff
@@ -8,7 +8,7 @@ freebsd_instance:
 task:
   name: FreeBSD 12.0
   env:
-    LOOM_MAX_DURATION: 10
+    LOOM_MAX_PREEMPTIONS: 2
```

Comment:
Are we going to set this back to 10? or did loom change so that 2 is enough to cover for most CI cases?

Reply:
They are very different measurements :) max preemptions of 2 is much better than max duration of 10

Reply:
Well good job to me I didnt even notice one was duration and one was preemptions ha!